### ISSUE TITLE
Restore ability for YANK to run implicit solvent-phase calculation on CPUs

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#83]
This was actually a very useful of old YANK: Implicit solvent phase calculations could be run efficiently on CPUs using the `CPU` or `Reference` platforms in parallel with the complex phase.

**Comment by jchodera** [comment]
This will be postponed to YANK 2.0.

**Comment by Lnaden** [comment]
Leave for post 1.0
